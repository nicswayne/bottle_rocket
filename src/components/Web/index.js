// @flow

import React from 'react';

import Footer from '../shared/Footer';

import forward from '../../resources/icons/ic_webForward@2x.png';
import back from '../../resources/icons/ic_webBack@2x.png';
import refresh from '../../resources/icons/ic_webRefresh@2x.png';

const Web = () => (
    <div>
        <div className="header">
            <div className="nav">
                <div className="button-group">
                    <button
                        className="button"
                        onClick={() => window.history.back()}
                    >
                        <img src={back} alt="" />
                    </button>
                    <button
                        className="button"
                        onClick={() => window.location.reload(true)}
                    >
                        <img src={refresh} alt="" />
                    </button>
                    <button
                        className="button"
                        onClick={() => window.history.forward()}
                    >
                        <img src={forward} alt="" />
                    </button>
                </div>
            </div>
        </div>
        <div className="web-body">
            <iframe
                src="https://www.bottlerocketstudios.com/contact"
                title="contact"
                sandbox="allow-scripts allow-top-navigation allow-forms allow-same-origin"
            />
        </div>
        <Footer />
    </div>
);

export default Web;
