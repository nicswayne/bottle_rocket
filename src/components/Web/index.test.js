import React from 'react';
import { shallow, render } from 'enzyme';

import Web from '.';

it('Web component should render without crashing.', () => {
    shallow(<Web />);
});

it('Web component should match snapshot.', () => {
    const wrapper = render(<Web />);

    expect(wrapper).toMatchSnapshot();
});
