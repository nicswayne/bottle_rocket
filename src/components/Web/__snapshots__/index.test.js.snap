// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Web component should match snapshot. 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="nav"
    >
      <div
        class="button-group"
      >
        <button
          class="button"
        >
          <img
            alt=""
            src="ic_webBack@2x.png"
          >
            
          </img>
        </button>
        <button
          class="button"
        >
          <img
            alt=""
            src="ic_webRefresh@2x.png"
          >
            
          </img>
        </button>
        <button
          class="button"
        >
          <img
            alt=""
            src="ic_webForward@2x.png"
          >
            
          </img>
        </button>
      </div>
    </div>
  </div>
  <div
    class="web-body"
  >
    <iframe
      sandbox="allow-scripts allow-top-navigation allow-forms allow-same-origin"
      src="https://www.bottlerocketstudios.com/contact"
      title="contact"
    >
      
    </iframe>
  </div>
  <div
    class="footer"
  >
    <a
      class="button"
      href="/"
    >
      <img
        alt=""
        src="tab_lunch@2x.png"
      >
        
      </img>
      <div
        class="text"
      >
        lunch
      </div>
    </a>
    <a
      class="button"
      href="/web"
    >
      <img
        alt=""
        src="tab_internets@2x.png"
      >
        
      </img>
      <div
        class="text"
      >
        internets
      </div>
    </a>
  </div>
</div>
`;
